lib/stdlib/test/shell_SUITE.erl

```diff
@@ -256,6 +256,8 @@ restricted_local(Config) when is_list(Config) ->
                   local_allowed(_,_,State) ->
                       {false,State}.
 
+                  non_local_allowed({erlang,raise},[_, _, _],State) ->
```

Comment:
Shouldn't it be sufficient to just let `erlang:raise(error, Reason, Stk)` through?

```suggestion
                  non_local_allowed({erlang,raise},[error, _, _],State) ->
```

Also, the documentation of restricted shell in the documentation of `shell` needs to be updated.